incubator/hnc/Makefile

```diff
@@ -90,12 +98,8 @@ test: build
 # Builds all binaries (manager and kubectl) and manifests
 build: generate fmt vet manifests
 	go build -o bin/manager ./cmd/manager/main.go
-	GOOS=linux GOARCH=amd64 go build \
-	     -o bin/kubectl/kubectl-hns_linux_amd64 \
-	     -ldflags="-X sigs.k8s.io/multi-tenancy/incubator/hnc/internal/version.Version=${HNC_IMG_TAG}" \
-	     ./cmd/kubectl/main.go
-	GOOS=darwin GOARCH=amd64 go build \
-	     -o bin/kubectl/kubectl-hns_darwin_amd64 \
+	GOOS=$(OS_NAME) GOARCH=amd64 go build \
+	     -o bin/kubectl/kubectl-hns_$(OS_NAME)_amd64 \
```

Comment:
I think we'd rather keep this as-is - every time we compile, we compile for multiple platforms so that we know if there's a problem early on. Also, we rely on this [in the release process](https://github.com/kubernetes-sigs/multi-tenancy/blob/775d0ca6bb7a9e1760be3eff46ad2eeffabba00f/incubator/hnc/cloudbuild.yaml#L28).

Reply:
Sure will revert this change.